dtfs-central-api/src/services/state-machines/fee-record/event-handlers/helpers/get-fixed-fee-for-facility.test.ts

```diff
@@ -120,11 +120,11 @@ describe('getFixedFeeForFacility', () => {
     // Act
     await getFixedFeeForFacility(facilityId, utilisation, reportPeriod);
 
-    const calculatedUtilisation = calculateUkefShareOfUtilisation(utilisation, aFacility().coverPercentage);
+    const ukefShareOfUtilisation = calculateUkefShareOfUtilisation(utilisation, aFacility().coverPercentage);
 
     // Assert
     expect(calculateFixedFee).toHaveBeenCalledWith<[CalculateFixedFeeParams]>({
-      utilisation: calculatedUtilisation,
+      utilisation: ukefShareOfUtilisation,
```

Comment:
Sorry by rename the function param I meant:
```suggestion
      ukefShareOfUtilisation: ukefShareOfUtilisation,
```

Reply:
Is this not out the scope of this ticket? by changing it here, I will need to change it in `calculateFixedFeeFromDaysRemaining` and every function that calls that, which means testing everything works again by Cherry etc.

Reply:
I mean this ticket is the ticket changing the meaning of this value so I don't see how that is out of scope. It's just changing the param name so should be low risk no? If it is not just a simple rename of the field on the type then ok to leave it but I think it should be?